### PR TITLE
[inductor] label cpp test files with oncall: cpu inductor

### PR DESCRIPTION
**Pull request by jgong5**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #115479
* __->__ #115167

cc @voznesenskym @penguinwu @EikanWang @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @peterbell10 @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @aakhundov @ColinPeppler

### CODE REVIEW

File: test/inductor/test_binary_folding.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Owner(s): ["module: inductor"]
+# Owner(s): ["oncall: cpu inductor"]
```
Comment (atalman) [inline test/inductor/test_binary_folding.py:1]: Looks like these tests have also some CUDA testing, so they are not only cpu:
https://github.com/pytorch/pytorch/pull/115167/files#diff-32177bda568814a1368e62144d5b08ad18bb92f5578d9994a5350c516d5e0294R35

Comment (atalman) [reply test/inductor/test_binary_folding.py]: @jgong5 Thank you!

Comment (jgong5) [reply test/inductor/test_binary_folding.py]: > @jgong5 Thank you! Please proceed as discussed on Slack by decoupling CPU and CUDA tests and assigning `# Owner(s): ["oncall: cpu inductor"]` to cpu tests

Thanks. Will do that in a separate PR.

**Comment by Le-Zheng** [comment]
> cc @Le-Zheng please help to check if any missing cpu-specific UT

Here are additional UT that may be cpu-specific:
test/Inductor/test_torchinductor.py

**Comment by jgong5** [comment]
> Here are additional UT that may be cpu-specific:
> test/Inductor/test_torchinductor.py: https://github.com/pytorch/pytorch/blob/main/test/inductor/test_torchinductor.py#L1645-L1656

@Le-Zheng Thanks. I think it is better to make these tests work for both cpu and cuda by replacing `"cpu"` with `self.device` instead of moving them to cpu repro.

**Comment by Le-Zheng** [comment]
> > Here are additional UT that may be cpu-specific:
> > test/Inductor/test_torchinductor.py: https://github.com/pytorch/pytorch/blob/main/test/inductor/test_torchinductor.py#L1645-L1656
> 
> @Le-Zheng Thanks. I think it is better to make these tests work for both cpu and cuda by replacing `"cpu"` with `self.device` instead of moving them to cpu repro. Can you submit a PR to make them device-agnostic?

Sure. I am creating a pull request to applying `self.device` https://github.com/pytorch/pytorch/pull/115519

**Review by atalman — APPROVED** [review]

**Pull request closed without merging**